source/rust_verify/src/rust_to_vir_base.rs

```diff
@@ -799,7 +799,7 @@ pub(crate) fn check_generic_bound<'tcx>(
     param_env_src: DefId,
     span: Span,
     trait_def_id: DefId,
-    args: &Vec<rustc_middle::ty::Ty<'tcx>>,
+    args: Option<&Vec<rustc_middle::ty::Ty<'tcx>>>, // None if it's a `trait A: B` generic bound on Self
```

Comment:
Is the `Option` still used?  It looks like it's always `Some`.

Reply:
Good catch, thanks! Forgot that the only user of None was the one I deleted.